data/pidgin/cpsAssets/23248703.json

```diff
@@ -571,7 +571,7 @@
   },
   "promo": {
     "headlines": {
-      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < >",
+      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < >",
```

Comment:
Hey, I know this is a draft (sorry for 👀 ) - why don't we replace the entire contents of this one with what's on the test CPS (https://www.test.bbc.com/pidgin/23248703.json)

Reply:
That probably makes a lot more sense but it's much less of a _statement_ piece than just removing the name